mantaray/colors.py

```diff
@@ -6,19 +6,19 @@
 # https://www.mirc.com/colors.html
```

Comment:
`_MIRC_COLORS` should be consistent with this link. But it doesn't mean that we have to use `_MIRC_COLORS` everywhere. For example:
- Instead of setting `1: "#242323"` in `_MIRC_COLORS`, please change `BACKGROUND = _MIRC_COLORS[1]` to `BACKGROUND = "#242323"` below.
- Instead of setting `10: "#e7b678"` in `_MIRC_COLORS`, change `textwidget.tag_configure("other-nick", foreground=_MIRC_COLORS[10], underline=True)` to `textwidget.tag_configure("other-nick", foreground="#e7b678", underline=True)`.

Reply:
Aren't mIRC colors just colors specific to that client? Why do you adhere to this color scheme at all?

Reply:
If I want to send `(:` where `:` is red, I want it to work!